integration-tests/fhir/src/main/java/org/apache/camel/quarkus/component/fhir/it/FhirDstu2RouteBuilder.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.fhir.it;
+
+import ca.uhn.fhir.context.FhirVersionEnum;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.fhir.FhirJsonDataFormat;
+import org.apache.camel.component.fhir.FhirXmlDataFormat;
+import org.apache.camel.quarkus.component.fhir.FhirFlags;
+
+public class FhirDstu2RouteBuilder extends RouteBuilder {
+
+    @Override
+    public void configure() {
+        Boolean dstu2 = !new FhirFlags.Dstu2Disabled().getAsBoolean();;
```

Comment:
s/;;/;/

Reply:
Could you please explain why are you doing things conditionally based on the value from the configuration? Is this maybe to be able to run the tests per protocol (because running them all is costly when debugging only one of them)?

Reply:
Explained in entension doc: adding a specification is costly, so fine tuning was added.